src/stmt.h

```diff
@@ -81,6 +84,7 @@ class ExprStmt : public Stmt {
     void Print(int indent) const;
 
     Stmt *TypeCheck();
+
```

Comment:
There's a bunch of unintended new empty line in this file. Would be good to remove.

Reply:
Changed